roles/mw/tasks/main.yml

```diff
@@ -0,0 +1,117 @@
+---
+# Setup mediawiki with Parsoid and fill it with content.
+
+- name: Install mediawiki
+  composer: command=install no_dev=True
+
+- name: Create wiki directories
+  shell: |
+    mkdir {{ ROOTDIR }}/tmp/sessions
+    mkdir {{ WIKIDIR }}/cache
+    mkdir {{ WIKIDIR }}/images/cache
+    mkdir {{ WIKIDIR }}/dumps
+
+- name: Set wiki folder permissions ({{ ROOTDIR }})
+  acl: entity={{ hw_owners }} path={{ ROOTDIR }} recursive=yes state=present #permissions=g+rw
+
+- name: Set wiki folder permissions ({{ ROOTDIR }}/tmp/sessions)
+  acl: entity={{ hw_owners }} path={{ ROOTDIR }}/tmp/sessions recursive=yes state=present permissions=ug+rw
+
+- name: Set wiki folder permissions ({{ WIKIDIR }}/images)
+  acl: entity={{ hw_owners }} path={{ WIKIDIR }}/images recursive=yes state=present permissions=ug+rw
+
+- name: Set wiki folder permissions ({{ WIKIDIR }}/cache)
+  acl: entity={{ hw_owners }} path={{ WIKIDIR }}/cache recursive=yes state=present permissions=ug+rw
+
+- name: Configure composer
+  copy: src={{ CONFDIR }}/composer.local.json dest={{ WIKIDIR }}/composer.local.json
+
+- name: Download basic MediaWiki extensions
+  composer: command=update no_dev=True working_dir={{ WIKIDIR }}
+
+- name: Point Mediawiki configuration to Hitchwiki configuration file
+  # Config file is stored elsewhere, require it from MW's LocalSettings.php
+  copy:
+    src: "{{ SCRIPTDIR }}/configs/mediawiki_LocalSettings.php"
+    dest: "{{ WIKIDIR }}/LocalSettings.php"
```

Comment:
Let's keep this as a symlink so that we can pull modifications to configuration file `/configs/mediawiki_LocalSettings.php` without needing to go re-copy it to `public/wiki/LocalSettings.php`.

Reply:
Oh wait, that file just import/requires it? So ignore what I just said. :-)

Reply:
done